addons/mail/static/src/models/thread/thread.js

```diff
@@ -458,6 +459,50 @@ function factory(dependencies) {
             this.update({ areAttachmentsLoaded: true });
         }
 
+        /**
+         * Fetch suggested recipients from the backend
+         */
+        async fetchAndUpdateSuggestedRecipients() {
+            const { [this.id]: data } = await this.async(() => this.env.services.rpc({
+                route: '/mail/get_suggested_recipients',
+                params: {
+                    model: this.model,
+                    res_ids: [this.id],
+                },
+            }));
+            if (!data) {
+                return;
+            }
+
+            const recipients = data.map(data => {
+                const parsedEmail = data[1] && mailUtils.parseEmail(data[1]);
+                const recipient = {
+                    checked: data[0] ? true : false,
+                    partner_id: data[0] ? data[0] : undefined,
+                    reason: data[2],
+                };
+                if (recipient.partner_id) {
+                    recipient.partner = [
```

Comment:
This is confusing, recipient does not have a partner_id field.
All of this should be recipient convertData actually.